app/assets/stylesheets/application.scss

```diff
@@ -122,8 +110,7 @@ h5 {
   background-color: $primary-color;
   color: $white;
   padding: 10px;
-  margin: 0px;
-  margin-top: 15px;
+  margin: 15px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `15px 0 0` instead of `15px 0 0 0`